diplomacy/web/src/gui/forms/message_form.jsx

```diff
@@ -320,14 +326,23 @@ static countries = [
     }
 
     generateSupportMoves(){
+
+        const { startLocation } = this.state;
+        const { engine } = this.props;
+
+        const possibleOrdersForGivenStart = get(engine, `possibleOrders[${startLocation}]`, []);
```

Comment:
This might have been the bug (or one of the occurrences in this PR where we try to map through an undefined varuable- using `[]` as fallback instead).